universe/Effect.cpp

```diff
@@ -3250,6 +3258,8 @@ void GenerateSitRepMessage::Execute(const ScriptingContext& context) const {
         }
     }
 
+    raw_sitrep_template = raw_sitrep_template && m_affiliation == AFFIL_SELF;
```

Comment:
Sounds reasonable to only send to the player that creates them... but in an MP game none of these techs would exist on the server anyway.

And regardless, why does that mean you want to adjust the raw_sitrep_template variable depending on the affiliation...?